avoinspector/src/main/java/app/avo/inspector/VisualInspectorMode.java

```diff
@@ -0,0 +1,6 @@
+package app.avo.inspector;
+
+public enum VisualInspectorMode {
+    bar,
+    bubble
+}
```

Comment:
No newline at the end of the file

Reply:
Also `enum` variables should be in UPPERCASE: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html#:~:text=Because%20they%20are%20constants%2C%20the,fields%20are%20in%20uppercase%20letters.&text=You%20should%20use%20enum%20types,a%20fixed%20set%20of%20constants.